.github/workflows/serinus_openapi.yml

```
@@ -0,0 +1,51 @@
+      - "packages/serinus_openapi/test/**"
+      - "packages/serinus_openapi/pubspec.yaml"
+
+jobs:
+  build:    
+    defaults:
+      run:
+        working-directory: ./packages/serinus_openapi
+    runs-on: "ubuntu-latest"
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)